colabs/stylegan_nada/StyleGAN-NADA.ipynb

```diff
@@ -0,0 +1,658 @@
+{
```

Comment:
<p>Maybe splitting in more cells. So if people re run the notebooks don't need to wget and re install everything</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/wandb/examples/pull/253/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>